### ISSUE TITLE
Error deleting note when Maps app is installed

### ISSUE THREAD

**Issue by xMartin** [nextcloud/notes#923]
Cannot delete a note when Maps app is installed. Tried the same after removing the Maps app and it worked fine.

## Steps to reproduce
1. Install Maps app (if you haven't already)
2. Create a note
3. Delete note from the context menu next to the note's title in the side bar

### Expected behaviour
Note is removed without an error and not shown even after refreshing the page.

### Actual behaviour
Note is removed but an error occurs. After refreshing the page the note appears again.

Error: "Deleting note XXX has failed. HTTP 500 (undefined)"
Logged error: `Doctrine\DBAL\Exception\TableNotFoundException: An exception occurred while executing a query: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'xxx_nextcloud.oc_maps_photos' doesn't exist`

## Server
Please complete the following information.

- Notes app version: 4.5.1
- Nextcloud version: 24.0.5
- OS: Linux
- Web server: 
- PHP version: 8.0.23
- Database: MySQL 10.3.22

**Nextcloud configuration:**

<details>
```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder
```
</details>

## Client
Please complete the following information.

- Browser (incl. version): Firefox 105
 - OS: mac os

## Log files
Add relevant parts of your `nextcloud.log` and/or your browser's JavaScript console here.

**Comment by korelstar** [comment]
Looks like this is a known issue from the maps app: https://github.com/nextcloud/maps/issues/649 - please try this manual fix: https://github.com/nextcloud/maps/pull/568#issuecomment-803649323

**Comment by xMartin** [comment]
> Looks like this is a known issue from the maps app: [nextcloud/maps#649](https://github.com/nextcloud/maps/issues/649) - please try this manual fix: [nextcloud/maps#568 (comment)](https://github.com/nextcloud/maps/pull/568#issuecomment-803649323)

That worked, thanks. Hope it gets fixed at some point.

I'm surprised this issue of another app can break the notes app. If that's by Nextcloud design and notes app can do nothing about it, this issue can be closed, I guess.

**Comment by korelstar** [comment]
> I'm surprised this issue of another app can break the notes app. If that's by Nextcloud design and notes app can do nothing about it, this issue can be closed, I guess.

Yes, indeed it's "by design" :-/
